pyproject.toml

```diff
@@ -45,6 +45,7 @@ jsonpickle = ">=2.2.0"
 qblox-instruments = "^0.14.0"
 lark-parser = "^0.12.0"
 pydantic-settings = ">=2.5.2"
+compiler-config = { git = "https://github.com/oqc-community/compiler-config.git", branch = "main" }
```

Comment:
This is a temporary solution until we have a pypi release of compiler-config with a specific version number.

Reply:
I'm happy with this in the interim